Doesn't Open!

### Describe the bug:
When I tried to run the file, it didn't work because there was a `NoClassDefFoundError`

### To Reproduce:
1. Download CItyManagement v0.8.1
2. Run `java -jar CityManagement-0.8.1.jar`

### Expected behavior:
The game should have started

### Screenshots:
![screen shot 2018-09-14 at 5 01 11 pm](https://user-images.githubusercontent.com/38357693/45574994-d6bd9800-b83f-11e8-8f73-fe41c4ff3465.png)


### Desktop (please complete the following information):
 - OS: macOS 10.13.1
 - Version: 0.8.1
 - Java vendor: Oracle
 - Java version: 1.8.0_161-b12

### Stack trace:
```
Exception in thread "main" java.lang.NoClassDefFoundError: com/gauck/sam/Utilities/Utilities
	at com.samgauck.CityManagement.Economy.<init>(Economy.java:14)
	at com.samgauck.CityManagement.Economy.getInstance(Economy.java:29)
	at com.samgauck.CityManagement.Command.<init>(Command.java:23)
	at com.samgauck.CityManagement.Command.getInstance(Command.java:46)
	at com.samgauck.CityManagement.Main.<clinit>(Main.java:44)
Caused by: java.lang.ClassNotFoundException: com.gauck.sam.Utilities.Utilities
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 5 more
```

### Additional context:
It happened as soon as I tried to start it.

--- a/build.gradle
+++ b/build.gradle
@@ -6,13 +6,6 @@ version '0.8.1'
 apply plugin: 'java'
 apply plugin: 'idea'
 
-jar {
-    manifest {
-        attributes("Manifest-Version": "1.0",
-        "Main-Class": "com.samgauck.CityManagement.Main")
-    }
-}
-
 sourceCompatibility = 1.8
 
 repositories {
@@ -23,6 +16,16 @@ repositories {
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compile "com.github.Samasaur1:Utilities:v+"
+}
+
+jar {
+
+    from configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
+
+    manifest {
+        attributes("Manifest-Version": "1.0",
+        "Main-Class": "com.samgauck.CityManagement.Main")
+    }
 }
 
 def incrementPatchVersion() {

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,6 @@
+#Sat Sep 15 08:04:36 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip

